boards/arm/adafruit_kb2040/adafruit_kb2040.dts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2021 Yonatan Schachter
```

Comment:
Same here

Reply:
Added a second line w/ my name and updated date.